Assets/Scripts/Combatants/Combatant.cs

```diff
@@ -99,6 +99,18 @@ public void AffectBodyPartByType(BodyPartType part, float value)
                 }
             }
         }
+
+        public void AddBodyPart(BodyPart bp)
+        {
+            _bodyPartsInventory.Add(bp);
+        }
+
+        //Damage on the body part -- Rin
+        public void AffectBodyPart(int index, float value)
```

Comment:
Would you mind renaming this to AffectBodyPartByIndex?

Reply:
Done!